src/configurations/nf/synapseConfigs/tools.ts

```diff
@@ -65,22 +65,277 @@ const tools: HomeExploreConfig = {
       name: 'Tools',
       cardConfiguration: toolsCardConfiguration,
       facetAliases,
-      searchConfiguration: {
-        searchable: [
-          'name',
-          'summary',
-          'studyName',
-          'fundingAgency',
-          'contact',
-          'type',
-          'subtype',
-          'diseaseFocus',
-          'manifestation',
-        ],
-      },
-
     },
   },
 }
 
+export const toolDetailsPageConfig: DetailsPageProps = {
+  sql: toolsSql,
+  sqlOperator: '=',
+  tabLayout: [
+    {
+      title: "Details",
+      synapseConfigArray: [
+        {
+          name: 'UserCardListRotate',
+          title: 'Tool Origin',
+          subtitle: 'Investigator',
+          props: {
+            sql: investigatorSql,
+            count: 1,
+            size: SynapseConstants.MEDIUM_USER_CARD,
+            useQueryResultUserData: true,
+            sqlOperator: '=',
+          },
+          tableSqlKeys: ['resourceId'],
+          columnName: 'resourceId',
+        },
+        {
+          name: 'CardContainerLogic',
+          title: 'Development Publication',
+          props: {
+            limit: 3,
+            facetAliases,
+            sql: developmentPublicationSql,
+            secondaryLabelLimit: 4,
+            ...publicationsV2CardConfiguration,
```

Comment:
Not a big deal, and it probably doesn't matter here (so feel free to reject), but prefer to spread shared config before individual properties to localize potential overrides

```suggestion
            ...publicationsV2CardConfiguration,
            limit: 3,
            facetAliases,
            sql: developmentPublicationSql,
            secondaryLabelLimit: 4,
```